com.parallax.PropellerIDE.json

```diff
@@ -0,0 +1,170 @@
+{
+    "app-id": "com.parallax.PropellerIDE",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "propelleride",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--device=all"
+    ],
```

Comment:
You should add a global or per-module `cleanup` array to remove things like development headers and static libraries.